src/libraries/Microsoft.PowerFx.Core/Utils/TextFormatUtils.cs

```diff
@@ -44,9 +44,10 @@ internal sealed class TextFormatUtils
         /// </summary>
         /// <param name="formatString">Raw input format string.</param>
         /// <param name="formatCulture">Current format culture.</param>
+        /// <param name="defaultLanguage">Default language.</param>
```

Comment:
>Default language. [](http://example.com/codeflow?start=42&length=17)

"default value for TextFormatArgs.FormatCultureName"

Reply:
update